app/atomic_file_test.py

```diff
@@ -14,6 +14,9 @@ class CallerError(Exception):
 class AtomicFileTest(unittest.TestCase):
 
     def setUp(self):
+        # Ignore pylint because we perform a tear down
```

Comment:
Our convention is to break lines at 80 columns. Can we make the first line extend to the 80-column mark before we add a newline to break it up?

(ditto throughout)